tableauserverclient/server/request_factory.py

```diff
@@ -129,6 +129,54 @@ def create_req(self, project_item):
         return ET.tostring(xml_request)
 
 
+class ScheduleRequest(object):
+    def create_req(self, schedule_item):
```

Comment:
These are getting so hard to read. I want to either put a comment above them which shows what the output of what we are building looks like or something.